### ISSUE TITLE
Multiple lookup servers

### ISSUE THREAD

**Issue by pastewka** [livMatS/dtool-lookup-gui#86]
We currently only support a single lookup server, but there are use cases where someone wanted to connect to multiple lookup servers. We could treat them on par with other data sources and enable the top search bar only when a lookup server data source is selected. Lookup servers of course cannot act as targets for copy operations.

**Comment by jotelha** [comment]
Tjelvar suggested at some point to introduce a "lookup storage broker" that implements the storage broker API only partially, that would make treating them on par with other base URIs easy.

**Comment by pastewka** [comment]
I agree, that works be the best solution and would simply the abstraction layer in the GUI.

________________________________
Von: "Johannes L. Hörmann" ***@***.***>
Gesendet: Freitag, 4. März 2022 09:26
An: livMatS/dtool-lookup-gui
Cc: Lars Pastewka; Author
Betreff: Re: [livMatS/dtool-lookup-gui] Multiple lookup servers (Issue #86) 

Tjelvar suggested at some point to introduce a "lookup storage broker" that implements the storage broker API only partially, that would make treating them on par with other base URIs easy.
